.envrc

```diff
@@ -0,0 +1 @@
+dotenv
```

Comment:
Nitpick, can you remove this file? I don't want to assume all contributors will have `dotenv`.

Reply:
I have no problem removing the file (used 2 commits for a reason), but I don't think it should affect people who don't use direnv/dotenv.

Reply:
@danielberkompas Removed this file.